target/linux/ath79/dts/qca9531_glinet_gl-xe300-nor-nand.dts

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "glinet,gl-xe300-nor-nand", "qca,qca9531";
+	model = "GL.iNet GL-XE300 Puli";
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "green:power";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		lan {
+			gpios = <0x0f 0x0a 0x01>;
+			label = "gl-xe300:green:lan";
+		};
+
+		wan {
+			gpios = <0x0f 0x01 0x01>;
+			label = "gl-xe300:green:wan";
+		};
+
+		wlan {
+			gpios = <0x0f 0x0c 0x01>;
+			label = "gl-xe300:green:wlan";
+			linux,default-trigger = "phy0tpt";
+		};
+
+		lte {
+			gpios = <0x0f 0x0d 0x01>;
+			label = "gl-xe300:green:lte";
+		};
+	};
+};
+
+&pcie0 {
+	status = "okay";
+};
+
+&usb0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	status = "okay";
+
+	hub_port: port@1 {
+		reg = <1>;
+		#trigger-source-cells = <0>;
+	};
+};
```

Comment:
Without a USB led, just status=okay is enough.

Reply:
You mean remove 'address-cells' and 'size-cells' keys?

Or simplify with:
&usb0 {
    status = "okay";
};

Reply:
The latter.

Reply:
Done, thanks.